VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -385,6 +388,21 @@ void qMRMLVirtualRealityViewPrivate::updateWidgetFromMRML()
     this->RenderWindow->SetDesiredUpdateRate(this->desiredUpdateRate());
   }
 
+  if (this->RenderWindow)
+  {
+    double PhysicalScale = this->MRMLVirtualRealityViewNode->GetPhysicalScale()*100;
+    if (PhysicalScale == 0)   
+    {  
+      PhysicalScale = 1;
+      this->RenderWindow->SetPhysicalScale(PhysicalScale);
+    }else{
```

Comment:
Brackets always in new lines in the current coding style please. Read here: https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide
Note that in this extension (and in Slicer too in the future) curly brackets align with the if/for/erc statement, not indented more as in current Slicer core.